From 0f36228b9a10bd31f3e8a1cc363ac3f0d56dc137 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mos=C3=A8=20Giordano?= <mose@gnu.org>
Date: Sun, 2 Jan 2022 15:36:12 +0100
Subject: [PATCH 1/1] Use lowercase name for `windows.h`

Case of file names matter on case-sensitive filesystems, and MinGW consistently
uses lowercase names for all header files and libraries.
---
 gdk/win32/gdkkeys-win32.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gdk/win32/gdkkeys-win32.h b/gdk/win32/gdkkeys-win32.h
index 14a69d8081..17b7d7125c 100644
--- a/gdk/win32/gdkkeys-win32.h
+++ b/gdk/win32/gdkkeys-win32.h
@@ -22,7 +22,7 @@
  */
 
 #include <glib.h>
-#include <Windows.h>
+#include <windows.h>
 
 /* For lookup table VK -> chars */
 typedef struct 
-- 
2.31.0

